src/model/implementation/root-entity-type.ts

```diff
@@ -168,6 +168,10 @@ export class RootEntityType extends ObjectTypeBase {
     validate(context: ValidationContext) {
         super.validate(context);
 
+        if (this.model.forbiddenRootEntityNames.find(value => this.name.toLowerCase() === value.toLowerCase())) {
+            context.addMessage(ValidationMessage.error(`RootEntities cannot be named ${this.name}.`, this.astNode));
```

Comment:
please use `this.nameASTNode` instead of `this.astNode`, otherwise it underlines the whole type definition including all fields which makes it pretty hard to read.